src/scss/abstracts/_variables.scss

```diff
@@ -2,23 +2,10 @@
   --black: #000000;
   --red: #e60012;
   --white: #ffffff;
-  --bluetitle:#577598;
```

Comment:
Are you sure all the variables you removed here, are not being used anywhere else?